frontend/components/buttonContants.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export const KeypadContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+`;
+
+export const Button = styled.button`
+  padding: 5px;
+  margin: 5px;
+  height: 100px;
+  width: 100px;
+  color: white;
+  font-size: 30px;
+  background-color: ${props => props.click ? '#1995AD' : '#A1D6E2'};
+`;
+
+export class ButtonContainer extends React.Component {
```

Comment:
Button Container is currently unnecessary, will delete once I get the disable functionality up and running